packages/assets-controllers/src/RatesController/multichain-rates/util.ts

```diff
@@ -0,0 +1,21 @@
+import { handleFetch } from '@metamask/controller-utils';
+
+import type { MarketDataDetailsFromPriceAPi } from './types';
+
+/**
+ * Fetches the rate for a given token on a specific chain.
+ *
+ * @param assetIds - An array of addresses of the tokens to fetch the rate for.
+ * @param includeMarketData - Whether to include market data in the response.
+ * @returns A promise that resolves to the rate of the token in USD.
+ */
+export function fetchMultiChainRates(
```

Comment:
this function will be removed and replaced with the one from snap API , but for now we work with mocked data